### PR TITLE
Fix LTP failures on newer distros

### PR DESCRIPTION
**Pull request by dsrivastavv**
LTp installation was failing during `make` on newer distro due to regression in the LTP version. Updated LTP version which fixed the issue

### CODE REVIEW

File: microsoft/testsuites/ltp/ltp.py [hunk 1]
```diff
@@ -315,12 +337,11 @@ def _install(self) -> bool:
...
         # tests, e.g, block_dev test
         make = self.node.tools[Make]
-        nprocs = self.node.tools[Nproc].get_num_procs()
         self.node.execute("autoreconf -f", cwd=ltp_path, sudo=True)
         make.make("autotools", cwd=ltp_path, sudo=True)
         self.node.execute("./configure --prefix=/opt/ltp", cwd=ltp_path, sudo=True)
-        make.make(f"-j {nprocs} all", cwd=ltp_path, sudo=True)
-        make.make(f"-j {nprocs} install SKIP_IDCHECK=1", cwd=ltp_path, sudo=True)
+        make.make("all", cwd=ltp_path, sudo=True)
+        make.make("install SKIP_IDCHECK=1", cwd=ltp_path, sudo=True)
```
Comment (LiliDeng) [inline microsoft/testsuites/ltp/ltp.py]: there is a method make_install with arguments, add comments for `SKIP_IDCHECK`

File: microsoft/testsuites/ltp/ltp.py [hunk 2]
```diff
@@ -50,7 +49,7 @@ class Ltp(Tool):
     _RESULT_LTP_ARCH_REGEX = re.compile(r"Machine Architecture: (.*)\s+")
 
     LTP_DIR_NAME = "ltp"
-    LTP_TESTS_GIT_TAG = "20190930"
+    LTP_TESTS_GIT_TAG = "20200930"
```
Comment (LiliDeng) [inline microsoft/testsuites/ltp/ltp.py:52]: please trigger testing against older and newer images, make sure there is no regression, thanks.

**Comment by dsrivastavv** [comment]
@LiliDeng Let's merge #2374 before merging this PR. I ran it on few newer distros and it passes the installation steps but the overall testcase fails due to copy back issue. Once #2374 is merged, I will ensure the tests are passing end to end.

**Comment by LiliDeng** [comment]
@somil55 #2374 has been merged, please rebase the code, and do some testing.

**Review by LiliDeng — APPROVED** [review]

**Pull request merged**